kork/kork-actuator/src/main/java/com/netflix/spinnaker/kork/actuator/observability/model/ArmoryEnvironmentMetadata.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2025 Harness, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.kork.actuator.observability.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ArmoryEnvironmentMetadata {
```

Comment:
The extra metadata is not needed for OSS as this was adding tags to keep track of the OSS metadata. Consider removing this or repurposing (although extra tags mean extra load for Observability platforms)